tests/e2e-tests/fixtures/fixture-loaders.js

```diff
@@ -249,6 +253,51 @@ const deleteShippingZones = ( ids ) => {
 	return Promise.all( ids.map( deleteZone ) );
 };
 
+const createBlockPages = () => {
+	return glob( `${ dirname( __filename ) }/../specs/**/*.fixture.json` )
+		.then( ( files ) => {
+			return Promise.all( files.map( ( file ) => readJson( file ) ) );
+		} )
+		.then( ( files ) => {
+			return Promise.all(
+				files.map( ( { title, pageContent: content } ) => {
```

Comment:
Seems like there might be some unnecessary additional looping here. Could you make the initial map an async call and thus only have to loop through the files once? Something like this:

```js
return Promise.all( files.map( async( file ) => { const file = await readJson( file ); /* rest of code here */ }
```

Reply:
I was trying to avoid adding async/await to this file so it can run while testing or directly via node or in other instances without needing to transpile it.
So basically there are two flat loops here (not nested loops), this is not much different from doing async as it has to wait for each file.

Reply:
I think you may be missing my point here. I'm not saying `createBlockPages` has to return a promise. I never pointed out nested loops I'm pointing out that your looping through the set of `files` _twice_. Once to get the JSON, and then again on all the JSON objects to do the request based on them. That's unnecessary imo I it looks to me like you could do this in **one** loop.